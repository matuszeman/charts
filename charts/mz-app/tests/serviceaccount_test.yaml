suite: serviceaccount
values:
  - ../values-ci.yaml
templates:
  - serviceaccount.yaml
tests:
  - it: should match snapshot with values-ci.yaml
    asserts:
      - matchSnapshot: {}
