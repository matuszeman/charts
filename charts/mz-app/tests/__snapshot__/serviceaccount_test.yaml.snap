should match snapshot with values-ci.yaml:
  1: |
    apiVersion: v1
    kind: ServiceAccount
    metadata:
      annotations:
        eks.amazonaws.com/role-arn: arn:aws:iam::123456789012:role/test-role
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: app-name
        app.kubernetes.io/version: 0.1.0
        helm.sh/chart: mz-app-0.1.0
      name: RELEASE-NAME-resource-suffix
